src/screens/feed/components/FeedPost/FeedPost.tsx

```diff
@@ -61,16 +73,26 @@ export const FeedPost = (props: FeedPostProps) => {
       borderTopRightRadius="lmin"
       marginTop="s">
       <FeedPostHeader post={post} openEditModal={openEditModal} borderColor={borderColor} />
-      <FeedPostBody
-        post={post}
-        borderColor={borderColor}
-        wasNavigatedFromNotifications={wasNavigatedFromNotifications}
-      />
-      <FeedPostFooter
-        {...props}
-        borderColor={borderColor}
-        isEditingTarget={editTarget?.type === 'comment' || editTarget?.type === 'post'}
-      />
+      {isPostBlocked ? (
+        <Box borderColor={borderColor} {...emptyBoxBorder} padding="m" alignItems="center">
+          <Text variant="textSM" textAlign="center">
+            {t('hiddenPostPlaceholder')}
+          </Text>
+        </Box>
+      ) : (
+        <>
+          <FeedPostBody
+            post={post}
+            borderColor={borderColor}
+            wasNavigatedFromNotifications={wasNavigatedFromNotifications}
+          />
+          <FeedPostFooter
+            {...props}
+            borderColor={borderColor}
+            isEditingTarget={!!(editTarget && ['comment', 'post'].includes(editTarget?.type))}
```

Comment:
```suggestion
            isEditingTarget={!!(editTarget && ['comment', 'post'].includes(editTarget.type))}
```
we can probably omit the `?` operator